hal/lib/athena/handles/HandlesInternal.h

```diff
@@ -23,6 +23,8 @@
 
 namespace hal {
 
+constexpr int16_t InvalidGetHandle = -1;
```

Comment:
Ok yeah that would work, especially now that the old constant that was named that has been removed.